x-pack/plugins/cases/public/components/add_file/index.tsx

```diff
@@ -25,20 +25,18 @@ import { FILE_ATTACHMENT_TYPE } from '../../../common/api';
 import { CASES_FILE_KINDS } from '../../files';
 import { useKibana } from '../../common/lib/kibana';
 import { useCreateAttachments } from '../../containers/use_create_attachments';
-import { useCasesContext } from '../cases_context/use_cases_context';
 import * as i18n from './translations';
 
 interface AddFileProps {
   caseId: string;
   onFileAdded: () => void;
+  owner: string;
 }
 
-const AddFileComponent: React.FC<AddFileProps> = ({ caseId, onFileAdded }) => {
+const AddFileComponent: React.FC<AddFileProps> = ({ caseId, onFileAdded, owner }) => {
```

Comment:
I'm not necessarily opposed to passing in the `owner` here but I think we typically do `useCasesContext()` to avoid prop drilling. Is the call to `useCasesContext` expensive? Just curious if we should break from that pattern and pass in `owner` and other context variables?

Reply:
I did it on purpose but I think I will just change back 😅 Normally I'd prefer using a context to avoid multilevel/complicated prop drilling and I thought here it looked nicer to fetch in one parent and pass to two children.

But I agree we should be consistent and it the code will look simpler.